package.json

```diff
@@ -13,7 +13,9 @@
     "url": "git://github.com/mcavage/node-asn1.git"
   },
   "main": "lib/index.js",
-  "dependencies": {},
+  "dependencies": {
+    "safer-buffer": "~2.1.0"
```

Comment:
This is not `^2.1.0` only because of npm bundled with Node.js 0.8 is too old and fails on ranges with `^`.

The code itself works on 0.8. I suggest changing to `~2.1.0` and dropping Node.js 0.8 from test matrix, but that could be done in a non-patch release, so I'm not including that to this PR.

Reply:
Yeah, I think we can drop 0.8.x from this now (pretty sure the last of our production systems on 0.8 at Joyent have moved over to 0.10 or 4.x now, but I'm still checking). We don't have the luxury of dropping 0.10 anytime soon from this library, though.

Reply:
I believe the only Joyent component on 0.8 still is https://github.com/joyent/sdc-amon/blob/master/Makefile#L34

Reply:
Ah, thanks @cburroughs. It looks like all of amon's deps that would pull in asn1 are pinned to a version that also pins asn1, so yeah, I think we can drop 0.8 compat in this library safely without breaking those images. Whenever we get around to updating it we'll probably have to update its node version with the other deps though.

Reply:
@arekinath good to merge and release then? :)

Reply:
Hey @arekinath, do you have an idea when we can expect the release to reach npm? It's one of the only two remaining dependencies we have to update for Yarn to stop printing the warning (https://github.com/yarnpkg/yarn/pull/6208) :)

Reply:
@arcanis see https://github.com/joyent/node-asn1/issues/32

Reply:
I've just published `asn1@0.2.4`.